docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+
+However, for authors of discrete articles, pieces of software or other research findings or work products, where names should be listed for any reason, this can be accommodated. This applies generally, to project developers and teams as well.
+
+### Simple English and plain language
+
+This is an important topic covered in the style guides listed below. See also [US Government resources on plain language](https://www.plainlanguage.gov/).
+
+Technical terminology alone -- both OSCAL terminology and the terminologies of its domains of application --  will sometimes make documentation difficult for some readers. But this is not a goal, and terminology aside, sentences should be kept short. When choosing between two ways to express the same thing, choose the simpler and commoner one.
+
+### Technical terminology
+
+When writing about OSCAL or topics related to OSCAL, technical terminology is indispensable and inescapable. Additionally, it is inevitable and foreseeable that some terms (examples that come readily to mind include <q>control</q>, <q>catalog</q>, <q>profile</q>, <q>constraint</q>, and <q>component</q>) will be overloaded in use, that is, they will be used in both technical and non-technical senses, sometimes within the same conversation. Moreover their various technical senses may sometimes be broadly *contradictory* (what is meant by <q>profile</q> might be specifically different from one technical context to another) without ever creating *ambiguity*, if their contexts of use are kept clear. Use technical terminology to prevent ambiguity, and where ambiguity arises, clarify context in order to clarify usage.
+
+For example, whenever writing about <q>validation</q>, be specific as to what kind of validation is meant, since the word has both generic and specific senses.
+
+Similarly, <q>control</q> has both an OSCAL sense, and an RMF sense (Risk Management Framework as defined by NIST SP800-37); a document that uses the term specifically in one of these senses, the other, or both (when OSCAL is being used to model RMF), might indicate this.
+
+Yet not every use of an overloaded term must be qualified. Various senses might be left implicit if no confusion is caused (again, considered in context). In accordance with a more general rule that plain common nouns are to be preferred over jargon, terms such as <q>control</q> or <q>profile</q> might be used without further explanation when context makes it clear enough what is meant *even if a term is also overloaded* (means other things in other contexts). Within the context of describing a processing pipeline for OSCAL profiles, <q>valid profile</q> might be taken as shorthand for <q>XML \[or JSON] document known to be valid using a \[profile] schema</q>, especially when this is stated outright at the top. The fact that other kinds of validation may be implicit, or not relevant, does not have to be explained.
```

Comment:
I like the sentiment, but I would disagree with this statement. Is this something we want to encourage in the style guide? I defer to the greater group.

When working on [our informational overview document regarding the emerging rules construct](https://hackmd.io/JZkedRyZRAmUEWOSxQn0jg) and the validation documentation in usnistgov/OSCAL#1169, this was one of the key issues we had in discussion with community: many misunderstandings in OSCAL occur due to the use of a common noun, and we presume the audience knows which meaning of the 1/`n` meanings of the overloaded common noun we mean.

Reply:
In view of your resistance I am making small edits on this paragraph. The advice is for me, actually (but if the shoe fits, wear it).